### PR TITLE
Added Edje syntax

### PR DESCRIPTION
**Pull request by sheosi**

### CODE REVIEW

**Comment by FichteFoll** [comment]
You are referencing the wrong repository.

**Comment by sheosi** [comment]
Sorry, I don't know what I did wrong, I mean, I'm pretty sure I followed the instructions as they are written.

File: repository/e.json [hunk 1]
```diff
@@ -220,6 +220,16 @@
 			]
 		},
 		{
+			"name": "Edje",
+			"details": "https://github.com/sheosi/package_control_channel",
```
Comment (joneshf) [inline repository/e.json]: This line should be:

```json
"details": "https://github.com/sheosi/sublime-edje-syntax",
```

**Comment by FichteFoll** [comment]
Also, You have to push a valid [semantic version](http://semver.org) [tag](http://git-scm.com/book/en/v2/Git-Basics-Tagging) (with optional leading `v`) to your repo.

**Comment by sheosi** [comment]
Ok, thank you so much :), I'll make another pull request.

**Pull request closed without merging**